batch-setup/make_meta_tiles.py

```diff
@@ -145,34 +175,28 @@ def missing_tiles_split(self, split_zoom, zoom_max, big_jobs):
         self.run_batch_job()
         tmpdir = tempfile.mkdtemp()
         try:
-            missing_meta_file = os.path.join(tmpdir, 'missing_meta.txt')
             missing_low_file = os.path.join(tmpdir, 'missing.low.txt')
             missing_high_file = os.path.join(tmpdir, 'missing.high.txt')
 
-            self.read_metas_to_file(missing_meta_file, compress=True)
-
             print("[make_meta_tiles] Splitting into high and low zoom lists")
 
             # contains zooms 0 until group zoom. the jobs between the group
```

Comment:
this comment is really confusing? the code below is saying `zoom_max` the comment here refers to `group zoom`, what is the relationship of these two terms?